spacy/schemas.py

```diff
@@ -405,13 +405,35 @@ class Config:
         arbitrary_types_allowed = True
 
 
+class ConfigSchemaDistillEmpty(BaseModel):
+    class Config:
+        extra = "forbid"
+
+
+class ConfigSchemaDistill(BaseModel):
+    # fmt: off
+    batcher: Batcher = Field(..., title="Batcher for the training data")
+    corpus: StrictStr = Field(..., title="Path in the config to the distillation data")
+    dropout: StrictFloat = Field(..., title="Dropout rate")
+    max_epochs: StrictInt = Field(..., title="Maximum number of epochs to distill for")
+    max_steps: StrictInt = Field(..., title="Maximum number of steps to distill for")
+    optimizer: Optimizer = Field(..., title="The optimizer to use")
+    pipe_map: Dict[str, str] = Field(..., title="Mapping from teacher to student pipe")
```

Comment:
The mapping is reversed in the [code](https://github.com/danieldk/spaCy/blob/e36234d70a0e981ed624bac9e0282847c7468ebc/spacy/language.py#L1048), i.e., student pipe -> teacher pipe. I think it'd be best to keep it the same in both the config and the code so as to avoid subtle bugs.

Reply:
👍 good catch. Also renamed this to `student_to_teacher` like @svlandeg suggested in the other PR.